perl/dist.ini

```diff
@@ -0,0 +1,48 @@
+name      = Gherkin
+abstract  = A parser and compiler for the Gherkin language
+main_module = lib/Gherkin.pm
+author    = ['Peter Sergeant <pete@clueball.com>', 'Gaspar Nagy']
+license   = MIT
+is_trial  = 0
+perl      = 5.008
+copyright_holder = Peter Sergeant, Gaspar Nagy
+
+; Worth seeing the note about this in Gherkin.pm. You are aiming for the Gherkin
+; version with the minor revision %2d and a two-digit CPAN version added. For
+; example:
+;
+;   3.1.2 -> 3.1.0201, 3.1.0202, 3.1.0203 ... etc
+;   3.2.0 -> 3.2.0001, 3.2.0002 ... etc
+;
+; CPAN releases can't be changed, and this allows a nice simple way of uploading
```

Comment:
I doubt we'll make emergency releases. We haven't in the past. We need a more streamlined release process now that there are 8 implementations of Gherkin.